client/src/components/commonComponents/NavBar/index.js

```diff
@@ -1,22 +1,62 @@
 import React from 'react';
-import { List, ListItem, Typography, Grid } from '@material-ui/core';
+import { List, ListItem, Typography, Grid, Box, Link } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import MenuIcon from '@material-ui/icons/Menu';
+import ShoppingCartOutlinedIcon from '@material-ui/icons/ShoppingCartOutlined';
 import logo from './logo.svg';
+import blog from './downearthblog.webp';
+
 import useStyles from './style.js';
 
 const NavBar = () => {
   const classes = useStyles();
   return (
     <div>
       <Grid container>
-        <Grid item xs={12} sm={12} className={classes.header}>
-          <Typography>Our Showroom is Open! (By Appointment Only)</Typography>
+        <Grid item xs={12} sm={12}>
+          <Typography className={classes.header}>
+            Our Showroom is Open! (By Appointment Only)
+          </Typography>
         </Grid>
       </Grid>
-      <div className={classes.logo}>
-        <img src={logo} className={classes.img} />
-        urban natural home
-      </div>
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        className={classes.subheader}
+      >
+        <Box p={1}>
+          <img src={blog} className={classes.blogImg} alt="blog" />
+        </Box>
+        <Box display="flex" justifyContent="flex-end" m={1} p={1}>
+          <Box p={1} display="flex">
+            <SearchIcon />
+            <Typography>Search</Typography>
+          </Box>
+          <Box p={1} display="flex">
+            <ShoppingCartOutlinedIcon />
+            <Typography>0 items</Typography>
+          </Box>
+        </Box>
+      </Box>
 
+      <Box
+        display="flex"
+        alignItems="center"
+        justifyContent="center"
+        className={classes.logo}
+      >
+        <img src={logo} className={classes.logoImg} alt="logo" />
+        urban natural home
+      </Box>
+      <Box
+        display="flex"
+        justifyContent="space-around"
+        className={classes.icons}
+      >
+        <MenuIcon />
+        <ShoppingCartOutlinedIcon />
+        <SearchIcon />
```

Comment:
when I click at menu icon it doesn't show the list! _( mobile view )_

Reply:
I fixed that. :sweat_smile: